### PR TITLE
Update Application Connector documentation

### PR DESCRIPTION
**Pull request by tomekpapiernik**
Changes proposed in this pull request:

- Minor fixes and improvements to wording. 
- CRD documents' names updated to follow the guidelines
- API specification links changed from relative to absolute

**Related issue(s)**
See #1283

### CODE REVIEW

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ The AC gives you the following:
 
 All of the AC components scale independently, which allows to adjust it to fit the needs of the implementation built using Kyma.
 
->**NOTE:** To learn more about the Environments in Kyma, read the **Environments** document in the **Kyma** documentation topic.
+>**NOTE:** To learn more about the concept Environments in Kyma, read the **Environments** document in the **Kyma** documentation topic.
```
Comment (bszwarc) [inline docs/application-connector/docs/001-overview-application-connector.md]: concept > concept of?

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
 title: Overview
 ---
 
-The Application Connector (AC) is a proprietary Kyma implementation that allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process doesn't change, which allows to avoid any configuration and network-related problems.
+The Application Connector (AC) is a proprietary Kyma implementation that allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process doesn't change, which allows to avoid any configuration or network-related problems.
```
Comment (bszwarc) [inline docs/application-connector/docs/001-overview-application-connector.md]: doesn't > does not

Comment (tomekpapiernik) [reply docs/application-connector/docs/001-overview-application-connector.md]: No need to be so strict about contractions. We allow them where they sound natural, that's the consensus ;)

Comment (bszwarc) [reply docs/application-connector/docs/001-overview-application-connector.md]: Sure, got it :)

**Pull request merged**